js/views/reviewsVw.js

```diff
@@ -0,0 +1,103 @@
+var __ = require('underscore'),
+    Backbone = require('backbone'),
+    $ = require('jquery'),
+    RatingCl = require('../collections/ratingCl'),
+    baseVw = require('./baseVw'),
+    ReviewView = require('./reviewVw');
+
+module.exports = baseVw.extend({
+
+  className: "flexCol-12 flex-border js-tabTarg js-reviews hide",
+
+  events: {
+  },
+
+  VIEWS_PER_BATCH: 25,
```

Comment:
This view implements scroll based lazy loading similar to what was done with the Blocked users tab in Settings. This implementation slightly differs as I was able to simplify a thing or two.